react-components/src/components/ViewerAnchor/ViewerAnchor.tsx

```diff
@@ -190,7 +190,7 @@ function computeCombinedTranslationCss(
   const inherentTranslation = `translateX(calc(${pointPositionOnScreen.x}px + ${stickyCompensation.x}px))
 translateY(calc(${pointPositionOnScreen.y}px + ${stickyCompensation.y}px))`;
 
-  const style = { ...inputStyle } ?? {};
+  const style = { ...inputStyle };
```

Comment:
Is this related to this PR? I didnt understand the purpose of this change 🤔

Reply:
No :see_no_evil: